src/equations/2d/ideal_glm_mhd.jl

```diff
@@ -453,14 +453,25 @@ function flux_hll(u_ll, u_rr, orientation, equation::IdealGlmMhdEquations2D)
   return SVector(f1, f2, f3, f4, f5, f6, f7, f8, f9)
 end
 
-
-# strong form of nonconservative flux on a side, e.g., the Powell term
-#     phi^L 1/2 (B^L+B^R) normal - phi^L B^L normal = phi^L 1/2 (B^R-B^L) normal
-# OBS! 1) "weak" formulation of split DG already includes the contribution -1/2(phi^L B^L normal)
-#         so this routine only adds 1/2(phi^L B^R normal)
-#         analogously for the Galilean nonconservative term
-#      2) this is non-unique along an interface! normal direction is super important
-function noncons_interface_flux(u_left, u_right, orientation, equation::IdealGlmMhdEquations2D)
+"""
+    noncons_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)
+
+Strong form of nonconservative flux on a side (Powell and GLM terms)
```

Comment:
```suggestion
Strong form of non-conservative flux on a surface (Powell and GLM terms)
```
"surfaces" is what we typically use in Trixi to denote anything "on the sides of an element", be it an interface (conforming), mortar (non-conforming), or boundary. But that's just a suggestion.